client/landing/stepper/declarative-flow/internals/steps-repository/free-setup/index.tsx

```diff
@@ -0,0 +1,111 @@
+import { StepContainer, base64ImageToBlob } from '@automattic/onboarding';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+import { useI18n } from '@wordpress/react-i18n';
+import React, { FormEvent, useEffect } from 'react';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { ONBOARD_STORE } from 'calypso/landing/stepper/stores';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { useSite } from '../../../../hooks/use-site';
+import SetupForm from '../components/setup-form';
+import type { Step } from '../../types';
+
+import './styles.scss';
+
+const FreeSetup: Step = function FreeSetup( { navigation } ) {
+	const { submit } = navigation;
+	const { __ } = useI18n();
+	const site = useSite();
+
+	const formText = {
```

Comment:
These placeholders don't seem to be respected see https://github.com/Automattic/wp-calypso/pull/70350#issuecomment-1331452659

Reply:
Resolved. I was actually still referencing the link in bio setup form, rather than the new free flow setup form.